ci/driver.sh

```diff
@@ -23,48 +23,62 @@ repo_url=${repo_url:-"https://github.com/NOAA-EMC/global-workflow.git"}
 ################################################################
 # Setup the reletive paths to scripts and PS4 for better logging 
 ################################################################
-set -eux
-pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+script_pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
 scriptname=$(basename "${BASH_SOURCE[0]}")
 echo "Begin ${scriptname} at $(date -u)" || true
 export PS4='+ $(basename ${BASH_SOURCE})[${LINENO}]'
 
-host=$(hostname) || true
-
 usage() {
   set +x
   echo
   echo "Usage: $0 -t <target> -h"
   echo
-  echo "  -t  target/machine script is running on    DEFAULT: ${host}"
-  echo "  -h  display this message and quit"
+  echo    "  -h  display this message and quit"
+  echo -n "  -t  target/machine script is running"
+  if [[ -n "${TARGET+x}" ]]; then
+    echo "on DEFAULT: ${TARGET}"
+  fi  
   echo
   exit 1
 }
 
+
+
 #########################################################################
 #  Set up runtime environment varibles for accounts on supproted machines
 #########################################################################
-TARGET=$(hostname) || true
-export TARGET
-while getopts "t:h" opt; do
-  case ${opt} in
-    t)
-      TARGET=${OPTARG}
-      ;;
-    h|\?|:)
-      usage
-      ;;
-    *)
-      echo "Unrecognized option" 
-      usage
-  esac
-done
+
+source "${script_pwd}/../ush/detect_machine.sh"
+if [[ "${MACHINE_ID}" != "UNKNOWN" ]]; then
+   TARGET="${MACHINE_ID}"
+fi   
+
+if [[ -z ${TARGET+x} || $1 == "-h" ]]; then
+  export TARGET
+  while getopts "t:h" opt; do
+    case ${opt} in
+      t)
+        TARGET=${OPTARG}
+        ;;
+      h|\?|:)
+        usage
+        ;;
+      *)
+        echo "Unrecognized option" 
+        usage
+    esac
+  done
+
+  if [[ -z "${TARGET+x}" ]]; then
+    usage
+  fi  
+
+fi
```

Comment:
I don't think we need this and the `usage` method anymore since we are using `detect_machine.sh` to detect the `TARGET`.  This script no longer needs to be provided a `TARGET` as input when called.

Reply:
Oh ok, I'll take it out then.